src/OpenLoco/src/Vehicles/Vehicle.h

```diff
@@ -73,6 +73,16 @@ namespace OpenLoco::Vehicles
     };
     OPENLOCO_ENABLE_ENUM_OPERATORS(SoundFlags);
 
+    enum UpdateVar1136114Flags : uint32_t
+    {
+        unk_m00 = (1 << 0),
+        noRouteFound = (1 << 1), // no route found??
+        crashed = (1 << 2),      // crashed??
+        unk_m03 = (1 << 3),
+        gradeCrossingSound = (1 << 4),
```

Comment:
This is just a general approaching grade crossing. Sure in the code we have implemented so far it's only for the sound but it has other users